patching/src/main/java/org/jboss/as/patching/ZipUtils.java

```diff
@@ -119,7 +119,7 @@ private static void unzip(final ZipFile zip, final File patchDir) throws IOExcep
             final String name = entry.getName();
             final File current = new File(patchDir, name);
             final String canonicalDestinationPath = current.getCanonicalPath();
```

Comment:
I am wandering if this variable definition could be removed and replaced by `current.getCanonicalPath()` when used in `throw PatchLogger.ROOT_LOGGER.entryOutsideOfPatchDirectory(canonicalDestinationPath);`

Reply:
Sounds good to me too